src/layout.js

```diff
@@ -2,7 +2,7 @@ import _ from 'lodash'
 
 import {
   run as acyclicRun,
-  run as acyclicUndo
+  undo as acyclicUndo
```

Comment:
Ha, nice find. I'm pretty sure I wasted a lot of time in the past trying to figure out what I broke. :+1: